src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            if (i > (figures.length - 1) / 2) {
+                figures[i] = figureSupplier.getDefaultFigure();
+            }
+            figures[i].draw();
+        }
+    }
```

Comment:
The 'main' method should not be static. Instead, create an instance of the Main class and call a non-static method that runs the application.